easybuild/easyconfigs/f/flex/flex-2.5.35-goolf-1.4.10.eb

```diff
@@ -8,7 +8,11 @@ sometimes called a tokenizer, is a program which recognizes lexical patterns in
 toolchain = {'name': 'goolf', 'version': '1.4.10'}
 toolchainopts = {'pic': True}
 
-sources = [SOURCELOWER_TAR_GZ]
-source_urls = ['http://prdownloads.sourceforge.net/%(namelower)s']
+sources = ['flex-2-5-35.tar.gz']
```

Comment:
@migueldiascosta please try to avoid hardcoding versions, it's bound to bite us in the ass later, via `--try-software-version` for example (which only updates the `version =` line)

```python
sources = ['flex-%s.tar.gz' % '-'.join(version.split('.'))]
```